src/ruby/spec/logconfig_spec.rb

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'spec_helper'
+require 'logger'
+
+describe GRPC do
+  describe '.logger=' do
+    it 'sets logger' do
+      GRPC.logger = GRPC::DefaultLogger::NoopLogger.new
```

Comment:
nit: instead of asserting the type of logger returned by `GRPC.logger`, can we just assert the object itself?

e.g.

```
noop_logger = GRPC::DefaultLogger::NoopLogger.new
GRPC.logger = noop_logger
expect(GRPC.logger).to be(noop_logger)
```

same for the stdout logger check too